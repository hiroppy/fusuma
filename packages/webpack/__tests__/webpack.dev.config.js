const dev = require('../src/webpack.dev.config');

describe('webpack.dev', () => {
  test('should match settings', () => {
    expect(dev()).toMatchSnapshot();
  });
});
