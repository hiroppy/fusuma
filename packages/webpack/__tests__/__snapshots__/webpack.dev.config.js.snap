// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`webpack.dev should match settings 1`] = `
Object {
  "devServer": Object {
    "contentBase": ".",
    "disableHostCheck": true,
    "hot": true,
    "inline": true,
    "port": 8080,
    "quiet": true,
  },
  "devtool": "inline-cheap-module-source-map",
  "mode": "development",
  "output": Object {
    "filename": "bundle.js",
  },
  "plugins": Array [
    FriendlyErrorsWebpackPlugin {
      "compilationSuccessInfo": Object {},
      "formatters": Array [
        [Function],
        [Function],
        [Function],
      ],
      "onErrors": undefined,
      "shouldClearConsole": true,
      "transformers": Array [
        [Function],
        [Function],
        [Function],
      ],
    },
  ],
}
`;
